src/Microsoft.Diagnostics.Repl/Microsoft.Diagnostics.Repl.csproj

```diff
@@ -10,7 +10,7 @@
     <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IncludeSymbols>true</IncludeSymbols>
-    <IsShippingAssembly>true</IsShippingAssembly>
+    <IsShippingPackage>false</IsShippingPackage>
```

Comment:
Wouldn't setting IsShippingAssembly to true mean that the package this is in doesn't ship but the assembly ships as part of another package/tool (dotnet-dump in this case)?

Reply:
`IsShippingAssembly` is true by default. If nothing is specified, everything but test projects are considered shipping packages and assemblies. I am just making sure the package ends up in transport.

Reply:
I think I see the difference now.  IsShipping is set to true by default (both by arcade and by the diagnostics repo).  IsShippingAssembly and IsShippingPackage are both set to IsShipping by default. So by setting IsShippingPackage to false, we get what we want: this assembly ships as part of dotnet-dump, but the package itself doesn't ship.